server/tests/api/test_world.py

```diff
@@ -107,9 +106,6 @@ def test_world_no_delete(client, world):
 
 
 @pytest.mark.django_db
-@pytest.mark.skipif(
-    settings.REDIS_USE_PUBSUB, reason="asyncio weirdness makes this fail"
-)
```

Comment:
But… now the text will never be skipped and always fail? It looks like we can'T run this test on pubsub? :(